src/components/Layout/NavBar/NavMenu.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <div class="nav-menu">
+    <div v-for="(menuItem, index) in menuData" :key="index" class="menu-item">
+      <router-link :to="menuItem.route">
+        <div class="icon">
+          <i class="fa" :class="menuItem.faClass"></i>
+        </div>
+        <span>{{ menuItem.label }}</span>
+      </router-link>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'NavMenu',
+  data() {
+    return {
+      sidebarOpen: false,
```

Comment:
i dont believe we have a sidebar anymore, although i dont think the current nav design will scale very well with more options. the header with the app logo right next to the menu icon looks out of place and forced. either the logo or the menu should be moved toe the opposite side of the header bar.